File: Day 1/twoSum-leetcode.cpp
#include <stdio.h>
#include <iostream>
#include <vector>
using namespace std;
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ves;
        for (int i=0;i<sizeof(nums);i++){
            for (int j=i; j<sizeof(nums);j++){
                if((nums[i]+nums[j])==target){
                    ves.push_back(i);
                    ves.push_back(j);
                    
                    return ves;
                }
                
            }
        }
        

    return ves;
} 
};

File: Day 1/twosum-Leetcode.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict={}
        for i in range(0,len(nums)):
            m=target-nums[i]
            dict[m]=i
           
        for i in nums:
            
            if i in dict:
                if (nums.index(i)!=dict.get(i)):
                    return ([nums.index(i),dict.get(i)])
